app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide :title, t(".title") %>
+<h1><%= t(".title") %></h1> 
+<p>
```

Comment:
Chỗ này nghiên cứu  Cách truyền tham số vào cho mesage Khi dùng tham số thì mình thêm các param dạng %{tên_param} vào message như sau:
https://viblo.asia/p/i18n-in-rails-Az45bbA65xY
để xử lý em nhé....k hiểu có thể hỏi anh